lib/ex_doc/formatter/html/autolink.ex

```diff
@@ -106,7 +106,7 @@ defmodule ExDoc.Formatter.HTML.Autolink do
                 language <- [:elixir, :erlang],
                 kind <- [:module, :function] do
               {kind, language, link_type}
-            end)
+            end) ++ [{:mix_task, :elixir, :normal}]
```

Comment:
I guess we could also support `:custom` links.
such as we do with the rest of the funtions.

https://github.com/elixir-lang/ex_doc/blob/5e00c0858aecdfa9506774316218162af25cd3f9/test/ex_doc/formatter/html/autolink_test.exs#L433-L434

```elixir
      assert project_doc("[the :erlang.apply/2 function](`Kernel.apply/2`)", %{}) ==
               "[the :erlang.apply/2 function](#{@elixir_docs}elixir/Kernel.html#apply/2)"
```

Reply:
@fertapric [convinced me](https://github.com/elixir-lang/ex_doc/issues/909#issuecomment-432937018) that custom links can sometimes hurt IEx experience so how about we hold off on this change for now?

Reply:
> @fertapric [convinced me](https://github.com/elixir-lang/ex_doc/issues/909#issuecomment-432937018) that custom links can sometimes hurt IEx experience so how about we hold off on this change for now?

Is that something that we could improve in IEX?